processor/manager_test.go

```diff
@@ -152,19 +150,16 @@ func TestProcessorManager(t *testing.T) {
 	mockRsourcesService := rsources.NewMockJobService(mockCtrl)
 
 	RegisterTestingT(t)
-	triggerAddNewDS := make(chan time.Time)
-	maxDSSize := 10
+	config.Reset()
+	c := config.New()
+	c.Set("JobsDB.maxDSSize", 10)
 	// tempDB is created to observe/manage the GW DB from the outside without touching the actual GW DB.
-	tempDB := jobsdb.Handle{
-		MaxDSSize: &maxDSSize,
-		TriggerAddNewDS: func() <-chan time.Time {
-			return triggerAddNewDS
-		},
```

Comment:
how are we overriding jobsdb now so that it doesn't trigger the addNewDS loop?

Reply:
It wasn't being used anywhere in the test.

Reply:
Could it be that the test wanted to simply disable this trigger?

Reply:
true
didn't think of it that way😅
Fortunately this didn't affect this particular test.
But in the future with the config dependency, we could set the `addDSLoop-timer` to a sufficiently large duration.